Sources/CvCityAI.cpp

```diff
@@ -8475,13 +8477,13 @@ void CvCityAI::AI_updateBestBuild()
 					}
 				}
 
-				if (aiFinalYields[YIELD_FOOD] >= iFoodPerPop)
+				if (aiFinalYields[YIELD_FOOD] >= iFoodPerPop) // if total food yield (on all plots?) is higher than food needed for 1 plot
```

Comment:
No, this is only the yield for the current plot in the plot loop.
So `if (plot has equal or more food yield than 1 population consume) { do something; }`